projects/igniteui-angular/src/lib/grids/grid/grid-filtering.spec.ts

```diff
@@ -649,6 +655,18 @@ describe('IgxGrid - Filtering actions #grid', () => {
         expect(grid.rowList.length).toEqual(1);
     }));
 
+    it('should exclude null and undefined values when filter by \'false\' and column dataType is not set', fakeAsync(() => {
+        fix = TestBed.createComponent(IgxGridESFComponent);
```

Comment:
Since the test is not using the UI we can use the `IgxGridFilteringComponent` instead of adding new component.
```suggestion
        fix = TestBed.createComponent(IgxGridFilteringComponent);
```